AlphaWallet/Common/Types/AppStyle.swift

```diff
@@ -82,9 +82,9 @@ extension UITabBarAppearance {
     }
 }
 
-extension UITabBarController {
-    static func withOverridenBarAppearence(appearence tabBarAppearance: UITabBarAppearance = .defaultAppearence) -> UITabBarController {
-        let tabBarController = UITabBarController()
+extension TabBarController {
```

Comment:
This should still work even if we don't introduce `TabBarController`, right?

Reply:
i was facing an issue when we do a swipe on browser tab its gonna to be swiped up to wallets screen, updating TabBar helps to solve that.